examples/GUI/GuiTools.js

```diff
@@ -25,58 +25,64 @@ dat.GUI.prototype.hideFolder = function hideFolder(name, value) {
     this.gui.domElement.id = domId;
     viewerDiv.appendChild(this.gui.domElement);
     this.colorGui = this.gui.addFolder('Color Layers');
     this.elevationGui = this.gui.addFolder('Elevation Layers');
 
     if (view) {
-        view.addEventListener('layers-order-changed', () => {
-            for (const layer of view.getLayers(l => l.type === 'color')) {
-                this.removeLayersGUI(layer.id);
+        view.addEventListener('layers-order-changed', (function refreshColorGui() {
+            var i;
+            var colorLayers = view.getLayers(function filter(l) { return l.type === 'color'; });
+            for (i = 0; i < colorLayers.length; i++) {
+                this.removeLayersGUI(colorLayers[i].id);
             }
 
-            const colorLayers = view.getLayers(l => l.type === 'color');
             this.addImageryLayersGUI(colorLayers);
-        });
+        }).bind(this));
     }
 }
 
 GuiTools.prototype.addImageryLayerGUI = function addImageryLayerGUI(layer) {
     var folder = this.colorGui.addFolder(layer.id);
-    folder.add({ visible: true }, 'visible').onChange((value) => {
+    folder.add({ visible: true }, 'visible').onChange((function updateVisibility(value) {
         layer.visible = value;
         this.view.notifyChange(true);
-    });
-    folder.add({ opacity: 1.0 }, 'opacity').min(0.0).max(1.0).onChange((value) => {
+    }).bind(this));
+    folder.add({ opacity: 1.0 }, 'opacity').min(0.0).max(1.0).onChange((function updateOpacity(value) {
         layer.opacity = value;
         this.view.notifyChange(true);
-    });
-    folder.add({ frozen: false }, 'frozen').onChange((value) => {
+    }).bind(this));
+    folder.add({ frozen: false }, 'frozen').onChange((function updateFrozen(value) {
         layer.frozen = value;
         this.view.notifyChange(true);
-    });
+    }).bind(this));
 };
 
 GuiTools.prototype.addElevationLayerGUI = function addElevationLayerGUI(layer) {
     var folder = this.elevationGui.addFolder(layer.id);
-    folder.add({ frozen: false }, 'frozen').onChange((value) => {
+    folder.add({ frozen: false }, 'frozen').onChange(function refreshFrozenGui(value) {
         layer.frozen = value;
     });
 };
 
 GuiTools.prototype.addImageryLayersGUI = function addImageryLayersGUI(layers) {
-    const seq = itowns.ImageryLayers.getColorLayersIdOrderedBySequence(layers);
-
-    for (const layer of layers.sort((a, b) => seq.indexOf(a.id) < seq.indexOf(b.id))) {
-        this.addImageryLayerGUI(layer);
+    var i;
+    var seq = itowns.ImageryLayers.getColorLayersIdOrderedBySequence(layers);
+    var sortedLayers = layers.sort(function comp(a, b) {
+        return seq.indexOf(a.id) < seq.indexOf(b.id);
+    });
+    for (i = 0; i < sortedLayers.length; i++) {
+        this.addImageryLayerGUI(sortedLayers[i]);
     }
 };
 
 GuiTools.prototype.addElevationLayersGUI = function addElevationLayersGUI(layers) {
-    for (var i = 0; i < layers.length; i++) {
+    var i;
```

Comment:
We have a linter rule for that. (`var-on-top` or something like this)

It's annoying, but useful too, as this `i` will get hoisted anyway.